frontend/admin/src/js/components/user/UserProfilePrintButton.tsx

```diff
@@ -0,0 +1,78 @@
+import { Button } from "@common/components";
+import { commonMessages } from "@common/messages";
+import React, { useRef } from "react";
+import { useIntl } from "react-intl";
+import { useReactToPrint } from "react-to-print";
+import { Scalars, useGetUserProfileQuery } from "../../api/generated";
+import UserProfileDocument from "./UserProfileDocument";
+
+export interface UserProfilePrintButtonProps {
+  userId: Scalars["ID"];
+}
+
+export const UserProfilePrintButton: React.FunctionComponent<{
+  userId: Scalars["ID"];
+}> = ({ userId, children }) => {
+  const intl = useIntl();
+
+  // would be nice to only fire this if the button was clicked but you would need an imperative version of the query that you could await on
+  const [{ data: initialData, fetching, error }] = useGetUserProfileQuery({
+    variables: { id: userId },
+  });
+
+  const componentRef = useRef(null);
+  const handlePrint = useReactToPrint({
+    content: () => componentRef.current,
+    pageStyle: `
```

Comment:
I'd like to centralize some standard print styling in a file but I'm not sure how to get CSS into a string. I tried CSS modules but that provides an object. I starting looking into some webpack plugins to inline a file but decided it was more trouble than was worth.

Reply:
You _could_ just export it as a string from a `.ts` file. 🤔 

`styles.ts`

```
export default `
@page {
  size: letter portrait;
}

@media print {
  .page-section {
    margin-bottom: 2rem;
    display: block;
    page-break-after: auto;
    page-break-inside: avoid;
    -webkit-region-break-inside: avoid;
  }
`
```

Reply:
Sometimes the simplest solutions are best.  Thanks Eric!  Added them in https://github.com/GCTC-NTGC/gc-digital-talent/pull/2967/commits/a9901301b0ca671dff173fc2096ccc43aca868d8.